src/app.js

```diff
@@ -1,11 +1,13 @@
-const express = require('express');
+const express = require("express");
+
+const artistRouter = require("./routes/artist");
+const albumRouter = require("./routes/album");
 
 const app = express();
 
 app.use(express.json());
 
-app.get("/", (_req, res) => {
-  res.send("Hello World!");
-});
+app.use("/", artistRouter);
+app.use("/", albumRouter);
```

Comment:
The base routes here should be /artists and /albums. Then you don't need to add them to the routes in the router files (see comments in those files too). i.e.
```
app.use("/artists", artistRouter);
app.use("albums", albumRouter);
```